packages/cli/__tests__/zosfiles/__system__/create/ds/__scripts__/create_ds_help.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+set -e
+
+echo "================Z/OS FILES CREATE DATA-SET HELP==============="
+zowe files create data-set my.data.copy --like my.data.orig --lrecl 1024 --help
```

Comment:
Please remove the arguments "my.data.copy --like my.data.orig --lrecl 1024". Only the "--help" argument should be passed for this test.